### ISSUE TITLE
Cannot get reasonable landmarks on 2D face alignment

### ISSUE THREAD

**Issue by tyshiwo** [hzh8311/pyhowfar#2]
Hi,

We have tried your code on 3D face alignment, which works fine. However, when you change the 3D coordinates to the 2D coordinates while keeping the codes and networks the same, we cannot achieve reasonable landmarks and the results are randomly distributed. Any suggestions may be helpful. Thanks!

**Comment by hzh8311** [comment]
Did you use `--pointType 2D` to conduct the 2D face alignment experiment？ If so, it should be comparable with the 3D face alignment.

**Comment by tyshiwo** [comment]
Hi,

Yes, we changed to '2D'  but failed to converge. Have you ever trained the model for 2D face alignment? Can you provide the pre-trained model?

**Comment by hzh8311** [comment]
Yes, I trained a 2D face alignment model, and get the validation curve below (left: 2D):
![validation curve](https://user-images.githubusercontent.com/15308569/42991376-dec0afb2-8c37-11e8-9f94-5eaaf3aa019f.png). So it is not a problem to train 2D model.

**Comment by tyshiwo** [comment]
Hi,

Which dataset did you use for training? I have used 300W_LP, and changed the --pointType to 2D, and got the following log:
![111 png](https://user-images.githubusercontent.com/19333875/42991915-a38eeeb6-8c39-11e8-9237-61e95f12768e.png)

Is it normal or not? Thanks.
